test/parallel/test-http-outgoing-finish.js

```diff
@@ -65,12 +65,12 @@ function write(out) {
     });
   });
 
-  out.end(buf, function() {
+  out.end(buf, common.mustCall(function() {
     endCb = true;
     console.error(`${name} endCb`);
     process.nextTick(function() {
       assert(finishEvent, `${name} got endCb event before finishEvent!`);
       console.log(`ok - ${name} endCb`);
```

Comment:
If the `console.log()` is not part of the test, I'd prefer to remove it, or replace it with a `debuglog` statement.

Reply:
this is an existing test... I'd rather do as little as possible in these... if that's alright?